modules/vba_documents/spec/factories/upload_submissions.rb

```diff
@@ -20,7 +20,7 @@
   }.freeze
 
   factory :upload_submission, class: 'VBADocuments::UploadSubmission' do
-    guid { 'f7027a14-6abd-4087-b397-3d84d445f4c3' }
+    guid { SecureRandom.uuid }
```

Comment:
This allows a developer to create multiple `UploadSubmission` records from the factory in the same spec without generating a "guid not unique" error.